model/model.js

```diff
@@ -0,0 +1,86 @@
+let fs = require('fs')
+let View = require("../view/view.js")
+
+class Model {
+    constructor(
+        ){
+        this.ID = 0
+        this.nama = 2
+        this.task = 3
+        this.status = 4
+        console.log(this.nama)
+
+    }
```

Comment:
isi constructor pada model disesuaikan dengan isi data, misalnya this.ID = ID, this.nama = nama